### ISSUE TITLE
Hip Centering in data preprocessing

### ISSUE THREAD

**Issue by YuktiADY** [weigq/3d_pose_baseline_pytorch#46]
Hallo,

I have been using this repo for estimating 3d skeleton. Below is my question that i would appreciate if you can answer.

  1.  In the code this is this line which is doing hip centering.
    poses = poses - np.tile( poses[:,:3], [1, len(H36M_NAMES)] )
    poses_set[k] = poses
    Is this necessary to do hip centering on 3d skeleton for estimating 3d pose? Is the hip centering only on 3d skeleton not on 2d ?
Because i tried without hip centering it seems validation is increased to 5.2

Because i have been training this model on my custom dataset but my validation loss which is 0.45 is not decreasing even after the so much hyperparametertuning.
2. Also can you please tell me what is coorelation between 2d and 3d skeleton. What exactly is done with 3d skeleton.
3. Can you give your suggestion on what can i do to decrease my validation loss(0.45) and training loss(0.112)

Please give your valuable suggestion.
